rollbar-cli/ChangeLog.md

```diff
@@ -1,3 +1,13 @@
 # Changelog for rollbar-cli
 
+All notable changes to this project will be documented in this file.
+
+## [1.1.0] - 2024-05-28
+
+### Changed
+- Added support for GHC 9.4.7
```

Comment:
@DavidMazarro I believe that stating that we added support for GHC 9.4 should suffice; even if we are only testing with 9.4.7, it is highly unlikely that a patch upgrade will introduce a backward incompatible change.

Reply:
Just trying to reduce the effort required to keep these CHANGELOG files up to date.

Reply:
Good point, thanks for the suggestion Sebas 🙂